### PR TITLE
adding opus_infopankki

### PR DESCRIPTION
**Pull request by patil-suraj**
Adding opus_infopankki
http://opus.nlpl.eu/infopankki-v1.php

### CODE REVIEW

File: datasets/opus_infopankki/opus_infopankki.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+import itertools
+import os
+
+import datasets
+
+
+_CITATION = """\
+J. Tiedemann, 2012, Parallel Data, Tools and Interfaces in OPUS. \
+In Proceedings of the 8th International Conference on Language Resources and Evaluation (LREC 2012)
```
Comment (lhoestq) [inline datasets/opus_infopankki/opus_infopankki.py:27]: can you use a citation in bibtex format here please ?

File: datasets/opus_infopankki/README.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+- es
+- et
+- fa
+- fi
+- fr
+- ru
+- so
+- sv
+- tr
+- zh
```
Comment (lhoestq) [inline datasets/opus_infopankki/README.md:18]:
```suggestion
  ar-en:
  - ar
  - en
  ar-es:
  - ar
  - es
  ar-et:
  - ar
  - et
  ar-fa:
  - ar
  - fa
  ar-fi:
  - ar
  - fi
  ar-fr:
  - ar
  - fr
  ar-ru:
  - ar
  - ru
  ar-so:
  - ar
  - so
  ar-sv:
  - ar
  - sv
  ar-tr:
  - ar
  - tr
  ar-zh:
  - ar
  - zh
  en-es:
  - en
  - es
  en-et:
  - en
  - et
  en-fa:
  - en
  - fa
  en-fi:
  - en
  - fi
  en-fr:
  - en
  - fr
  en-ru:
  - en
  - ru
  en-so:
  - en
  - so
  en-sv:
  - en
  - sv
  en-tr:
  - en
  - tr
  en-zh:
  - en
  - zh
  es-et:
  - es
  - et
  es-fa:
  - es
  - fa
  es-fi:
  - es
  - fi
  es-fr:
  - es
  - fr
  es-ru:
  - es
  - ru
  es-so:
  - es
  - so
  es-sv:
  - es
  - sv
  es-tr:
  - es
  - tr
  es-zh:
  - es
  - zh
  et-fa:
  - et
  - fa
  et-fi:
  - et
  - fi
  et-fr:
  - et
  - fr
  et-ru:
  - et
  - ru
  et-so:
  - et
  - so
  et-sv:
  - et
  - sv
  et-tr:
  - et
  - tr
  et-zh:
  - et
  - zh
  fa-fi:
  - fa
  - fi
  fa-fr:
  - fa
  - fr
  fa-ru:
  - fa
  - ru
  fa-so:
  - fa
  - so
  fa-sv:
  - fa
  - sv
  fa-tr:
  - fa
  - tr
  fa-zh:
  - fa
  - zh
  fi-fr:
  - fi
  - fr
  fi-ru:
  - fi
  - ru
  fi-so:
  - fi
  - so
  fi-sv:
  - fi
  - sv
  fi-tr:
  - fi
  - tr
  fi-zh:
  - fi
  - zh
  fr-ru:
  - fr
  - ru
  fr-so:
  - fr
  - so
  fr-sv:
  - fr
  - sv
  fr-tr:
  - fr
  - tr
  fr-zh:
  - fr
  - zh
  ru-so:
  - ru
  - so
  ru-sv:
  - ru
  - sv
  ru-tr:
  - ru
  - tr
  ru-zh:
  - ru
  - zh
  so-sv:
  - so
  - sv
  so-tr:
  - so
  - tr
  so-zh:
  - so
  - zh
  sv-tr:
  - sv
  - tr
  sv-zh:
  - sv
  - zh
  tr-zh:
  - tr
  - zh
```

**Review by lhoestq — APPROVED** [review]
thank you !

**Comment by patil-suraj** [comment]
Thanks Quentin !

**Pull request merged**